src/prefect/engine/executors/dask.py

```diff
@@ -201,3 +202,77 @@ def wait(self, futures: Any) -> Any:
                 return client.gather(futures)
         else:
             raise ValueError("This executor has not been started.")
+
+
+class LocalDaskExecutor(Executor):
+    """
+    An executor that runs all functions locally using `dask` and a configurable dask scheduler.  Note that
+    this executor is known to occasionally run tasks twice when using multi-level mapping.
+
+    Args:
+        - scheduler (str): The local dask scheduler to use; common options are "synchronous", "threads" and "processes".  Defaults to "synchronous".
+        - **kwargs (Any): Additional keyword arguments to pass to dask config
+    """
+
+    def __init__(self, scheduler: str = "synchronous", **kwargs: Any):
+        self.scheduler = scheduler
+        self.kwargs = kwargs
+        super().__init__()
+
+    @contextmanager
+    def start(self) -> Iterator:
+        """
+        Context manager for initializing execution.
+
+        Configures `dask` and yields the `dask.config` contextmanager.
+        """
+        with dask.config.set(scheduler=self.scheduler, **self.kwargs) as cfg:
+            yield cfg
+
+    def queue(self, maxsize: int = 0) -> Queue:
```

Comment:
Not important for this PR, but are we using these queues anywhere? Can we kill them?

Reply:
I can kill them in this PR, we are not using them anywhere!

Reply:
Although maybe a user is? I'll add this to breaking changes...

Reply:
Removed.